src/pages/pt-BR/install/auto.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Instale Astro com a Interface de Linha de Comando Automática.
```

Comment:
Oh, one detail actually! I guess there should not be a `.` at the end of the title right?

```suggestion
title: Instale Astro com a Interface de Linha de Comando Automática
```